modules/check_in/app/services/v2/chip/service.rb

```diff
@@ -195,7 +195,7 @@ def confirm_demographics_id_params
 
         {
           patientDfn: hashed_identifiers[:patientDFN],
-          stationNo: hashed_identifiers[:stationNo]
+          stationNo: hashed_identifiers[:stationNo].to_s
```

Comment:
minor: not 100% sure if it's more idiomatic to do `"#{hashed_identifiers[:stationNo]}"`, will leave it to you.

Reply:
Sure, will check when we consolidate these methods used in refresh & confirm-demographics